library.js

```diff
@@ -19,10 +19,14 @@ const nbbAuthController = require.main.require(
   './src/controllers/authentication',
 );
 
-const userInfoUrl = 'https://auth.dataporten.no/openid/userinfo';
-const memberStatusUrl = 'https://api.dataporten.no/userinfo/v1/userinfo';
-const organizationalInfoUrl =
-  'https://groups-api.dataporten.no/groups/me/groups';
+// eslint-disable-next-line no-unused-vars
+const environment = process.env.NDLA_ENVIRONMENT;
+
+// eslint-disable-next-line no-template-curly-in-string
+const feiderUserUrl =
```

Comment:
Når eg tenker meg om går det kanskje an å bruke api-gateway.ndla-local på alle? Jaja. Skader ikkje at det er slik.

Reply:
Kan være det skader litt _dersom_  vi ikke har løst det problemet vi hadde med aws for _leeenge_ siden som vi egentlig aldri fant ut av.
I tillegg så vil vi vel alltid bruke både egress og ingress her (med mindre aws gjør noe smart internrouting som jeg ikke vet om).

Om mulig så burde vi nok bruke `API_GATEWAY_HOST` som kommer fra `common`